### PR TITLE
Add new percentage filtering doc

### PR DESCRIPTION
**Pull request by thompiler**
- Closes https://github.com/airbrake/airbrake-docs/issues/104

### Screenshot of new article
screen shot 2017-03-16 at 6 22 17 pm

### CODE REVIEW

File: jekyll/_docs/airbrake-faq/send-percentage-of-total-errors.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+description: how to send a percentage of the total errors
+---
+
+Sometimes you might want to reduce the total amount of errors a project reports.
+This could be helpful in cases where a specific project is reporting a large
+amount of errors and is contributing to the account reaching its
+[rate limit](https://airbrake.io/docs/airbrake-faq/what-is-a-rate-limit/) often.
+
+By only reporting a percentage of errors, you reduce the total errors reported
+while ensuring that the errors reported are an accurate sample of all the
```
Comment (mmcdaris) [inline jekyll/_docs/airbrake-faq/send-percentage-of-total-errors.md]: We use "report" a lot here, what do you think of this variation:
> By only reporting a percentage of errors, you'll hit your plan's rate limit less often while ensuring that you still receive an accurate sample of your project's errors.

Comment (thompiler) [inline jekyll/_docs/airbrake-faq/send-percentage-of-total-errors.md]: Sounds good to me. 👍

File: jekyll/_docs/airbrake-faq/send-percentage-of-total-errors.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+### Example in Ruby
+
+In this example, we'll ignore the same percentage of errors but we'll be using
+[our gem's `add_filter`
+method](https://github.com/airbrake/airbrake-ruby#airbrakeadd_filter).
+
+```rb
+Airbrake.add_filter do |notice|
+  # Generate random number between 1 and 3
+  random_number = Random.rand(3) + 1
```
Comment (kyrylo) [inline jekyll/_docs/airbrake-faq/send-percentage-of-total-errors.md]:
```rb
random_number = rand(1..3)

File: jekyll/_docs/airbrake-faq/send-percentage-of-total-errors.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+  # Ignore error if random number is 1
+  notice.ignore! if random_number  == 1
```
Comment (kyrylo) [inline jekyll/_docs/airbrake-faq/send-percentage-of-total-errors.md]: Extra space.

**Comment by kyrylo** [comment]
I am not sure what you suggest here is a good approach. Why would anyone rely on random numbers to ignore notices? This sounds dangerous.

**Comment by thompiler** [comment]
>I am not sure what you suggest here is a good approach. Why would anyone rely on random numbers to ignore notices? This sounds dangerous.

People have written in to support asking how to reduce their overall error traffic volume by a certain percentage (eg: 30%). This is what I recommended (not sure if it's important but they were using Javascript BTW). 

Could you elaborate on what is dangerous about using a random number to do this? If a user wants to reduce the total error traffic of a project by a certain percentage, what would you recommend?

**Comment by thompiler** [comment]
>I am not sure what you suggest here is a good approach. Why would anyone rely on random numbers to ignore notices? This sounds dangerous.

I'll remove the doc in the meantime too.

**Comment by joegod** [comment]
So, we added this to the doc and then removed it?  I think it's pretty valuable, and unless I'm missing something doesn't seem hokey or insecure.  The random number thing is weird if it's a low-volume error, but in that case there'd be no need to sample. And for high-volume, it's really the only way to do sampling, right?

@thompiler - side note: we should actually refer to this as "sampling".  Some of our competitors threaten to sample if you exceed certain thresholds, and we can just talk here about putting the power to sample in your hands.  Instead of us deciding which groups to sample, you can proactively sample high-volume groups of known errors and ensure that you get each instance for lower-volume and more critical errors.

**Pull request merged**